### DISCUSSION TITLE
Certain settings interfere with ReCaptcha verification?

### DISCUSSION THREAD

**Discussion by e-t-l** [sereneblue/chameleon#572] [Q&A]
I know issues #250 and #477 relate to Captcha errors, and it's implied that Chameleon doesn't ALWAYS break ReCaptcha, but certain Chameleon settings might. Can you help me figure out which settings those might be?

When I use the attached config, Google ReCaptcha will load and allow me to click images, but when I press the Verify button, it pops an error message "Cannot connect to ReCaptcha." Disabling Chameleon, even without reloading the tab, fixed this.

[chameleon_1697461834091.json](https://github.com/sereneblue/chameleon/files/12917145/chameleon_1697461834091.json)

**Comment by sereneblue** [comment]
Hi @e-t-l, 

Can you replicate this on https://google.com/recaptcha/api2/demo? Are you using a VPN?

I've tried using your settings with a new Firefox profile but couldn't replicate. Chameleon shouldn't be spoofing anything recaptcha.
